valqueries-automapper/src/main/java/com/valqueries/automapper/SqlGenerator.java

```diff
@@ -92,13 +90,24 @@ private SqlDescriber.DbIndex toDbIndex(KeySet index) {
 	}
 
 	public String generateCreateTable(TypeDescriber<?> typeDescriber) {
+		ValqueriesSchemaBuilder schemaBuilder = prepareSchemaBuilder(typeDescriber);
+		schemaBuilder.build();
+		return schemaBuilder.getGeneratedSql();
+	}
+
+	private ValqueriesSchemaBuilder prepareSchemaBuilder(TypeDescriber<?> typeDescriber) {
 		ValqueriesSchemaBuilder schemaBuilder = schemaBuilderProvider.get();
-		schemaBuilder.addTable(dialect.getTableName(Clazz.of(typeDescriber.clazz())), table -> {
+		schemaBuilder.addTable(getTableName(typeDescriber), table -> {
```

Comment:
Not sure why this was changed?

Reply:
The semantics are the same. `getTableName()` is an utilty method that makes it easier to read. It was already in the class ans used somewhere else so I just refactored the code that didn't use it as part of the pr. The tests looked a bit verbose to me.

Reply:
Ah, didn't realize it was a utility that did the same, :+1: